lab_1/main.py

```diff
@@ -25,7 +35,12 @@ def remove_stop_words(tokens: list, stop_words: list) -> list:
     stop_words = ['the', 'is']
     --> ['weather', 'sunny', 'man', 'happy']
     """
-    pass
+    if not isinstance(tokens, list):
+        return []
+    elif not isinstance(stop_words, list) or not stop_words:
+        return tokens
+    tokens1 = [i for i in tokens if i not in stop_words]
+    return tokens1
```

Comment:
you can return here `return [i for i in tokens if i not in stop_words]` directly